lib/views.js

```diff
@@ -1,25 +1,9 @@
-exports.facilities = {
-  map: function (doc) {
-    if (doc.type === 'clinic' ||
-        doc.type === 'health_center' ||
-        doc.type === 'district_hospital' ||
-        doc.type === 'national_office' ||
-        doc.type === 'person') {
+/**
+ * This file contains all the views to be bundled up in the "medic" ddoc.
+ * If you want a view to be bundled into the "medic-client" ddoc and therefore
+ * be available on the client via pouch add it to: /ddocs/medic-client/views
+ */
```

Comment:
No. All ddocs are available on the server so you can just request "medic-client/my_view" - there's no need to duplicate the view. Duplication would impact server performance as each change has to be run through each view.

Views should be added to "medic" if they only need to be accessed on the server.
Views should be added to "medic-client" if they need to be accessed on the client only, or the client and the server.